### PR TITLE
Small Fixes, Rewording, Visual Quirks

### PR DESCRIPTION
**Pull request by billdybas**
~~~@robmcl4 Add to this PR any of the small things you wanted to change.~~~

This PR includes:
- Rewording the Student Success Modal
- Fixing Caching Issues on Student Dashboard
- Moving Frontend URLs into Constants
- Fixing a Backend Regression
- Reskinning the Voting Interface to blend the buttons into the grey background (i.e. remove the white section)

### CODE REVIEW

**Comment by billdybas** [comment]
Tina, I don't know why you approved this. It's still in WIP.

**Comment by chtinahow** [comment]
I assumed it wasn't since it didn't have any WIP anywhere. my bad

**Comment by billdybas** [comment]
@robmcl4 I think all that's left on this PR is to finish the voting interface styling. It's good to merge one the styles are in.

**Comment by billdybas** [comment]
I think I'm also going to add in a constant for the image path URL so we can remove all the TODO comments.

File: frontend/src/Admin/components/NavBar.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default class NavBar extends Component {
                 <NavLink tag={Link} to='/' onClick={this.props.switchToJudge}>
                   View as Judge |
                 </NavLink>
+                {/* TODO: Move the | out of the link */}
```
Comment (chtinahow) [inline frontend/src/Admin/components/NavBar.js]: Does this TODO need to be removed?

Comment (billdybas) [reply frontend/src/Admin/components/NavBar.js]: Nope, at least not in this PR – just making a note here because Rob wanted to change this.

**Comment by billdybas** [comment]
@chtinahow The buttons have always looked that way because they are in a button group. I've determined at this point, it's not worth the effort to change. More important right now to get this PR merged.

**Pull request merged**